README.md

```diff
@@ -1209,14 +1209,22 @@ To enable Call Caching, add the following to your Cromwell [configuration](#conf
 
 ```
 call-caching {
-  enabled = true
+  enabled = false
+  hash-docker-names = true
   lookup-docker-hash = false
+  hash-file-contents = true
+  hash-file-paths = false
 }
 ```
 
-When `call-caching.enabled=true`, Cromwell will add completed calls to the cache as well as do a cache lookup before running any call.
+When `call-caching.enabled=true` (default: `false`), Cromwell will be able to to copy results from previously run jobs (when appropriate).
+
```

Comment:
@cjllanwarne To poke at your "flexibility" argument, if you want this stuff to be configurable, why stop here? Why not make every component able to be turned off?